app/models/levelling_up_premium_payments/eligibility.rb

```diff
@@ -36,6 +36,20 @@ class Eligibility < ApplicationRecord
       "itt_academic_year" => ["eligible_itt_subject"]
     }.freeze
 
+    # Generates an object similar to
+    # {
+    #   <AcademicYear:0x00007f7d874293c8 @start_year=2017, @end_year=2018> => "2017/2018",
+    #   <AcademicYear:0x00007f7d87429288 @start_year=2018, @end_year=2019> => "2018/2019",
+    #   <AcademicYear:0x00007f7d87429260 @start_year=2019, @end_year=2020> => "2019/2020",
+    #   <AcademicYear:0x00007f7d87429238 @start_year=2020, @end_year=2021> => "2020/2021",
+    #   <AcademicYear:0x00007f7d87429210 @start_year=2021, @end_year=2022> => "2021/2022",
+    #   <AcademicYear:0x00007f7d87428c98 @start_year=nil, @end_year=nil> => "None"
+    # }
+    SELECTABLE_ITT_ACADEMIC_YEARS =
+      JourneySubjectEligibilityChecker.selectable_itt_years_for_claim_year(AcademicYear.for(Date.today)).each_with_object({}) do |year, hsh|
```

Comment:
I think we need to get the academic year for the current `PolicyConfiguration`, rather than the current calendar academic year. E.g. `PolicyConfiguration.for(LevellingUpPremiumPayments).current_academic_year`